charts/posthog/templates/_helpers.tpl

```diff
@@ -350,6 +350,9 @@ Create the name of the service account to use
 {{- $info := set $info "hostname" .Values.ingress.hostname -}}
 {{- $info := set $info "operation" (include "posthog.helmOperation" .) -}}
 {{- $info := set $info "ingress_type" (include "ingress.type" .) -}}
+{{- if .Values.deploymentKind -}}
+{{- $info := set $info "deployment_kind" .Values.deploymentKind -}}
```

Comment:
kind is kind of a bad name - overloaded

I suggest `medium` with default to `helm`